ui/pages/settings/security-tab/delete-metametrics-data-button/delete-metametrics-data-button.tsx

```diff
@@ -97,7 +98,6 @@ const DeleteMetaMetricsDataButton: DeleteMetaMetricsDataButtonComponent =
           <Box
             ref={ref}
             className="settings-page__content-row"
-            data-testid="delete-metametrics-data-button"
```

Comment:
Remove as this is not a button, and in the test this selector is not very useful.